pkg/causality/conflict_detector.go

```diff
@@ -16,17 +16,19 @@ package causality
 import (
 	"sync"
 
+	"github.com/pingcap/log"
 	"github.com/pingcap/tiflow/pkg/causality/internal"
 	"github.com/pingcap/tiflow/pkg/chann"
 	"go.uber.org/atomic"
+	"go.uber.org/zap"
 )
 
 // ConflictDetector implements a logic that dispatches transaction
```

Comment:
// ConflictDetector takes in a series of transactions.
// It checks if these transactions depend on each other,
// and sends them to different workerCaches channels.
// Transactions in different channels can be executed concurrently.
// It ensures that transactions changing the same
// rows don't executed concurrently and keep their original order.